app/javascript/app/components/ghg-emissions/ghg-emissions-selectors/ghg-emissions-selectors-table-data.js

```diff
@@ -52,6 +52,8 @@ export const getTableData = createSelector(
       return { [String(d.x)]: formatValue(d[c.value]) }; // year: value
     };
     return yColumnOptions.map(c => ({
+      // Conditionally add ISO to the data when countries or regions is selected
+      ...(model === 'regions' && { iso: c.iso }),
```

Comment:
**Note:** the model is also `regions` when the user selects to show data by *Countries*